projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/graphql/model/DataJobDeployment.java

```diff
@@ -0,0 +1,270 @@
+package com.vmware.taurus.service.graphql.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.media.Schema.RequiredMode;
+import java.time.OffsetDateTime;
+import java.util.Objects;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.format.annotation.DateTimeFormat.ISO;
+
+@Schema(name = "DataJobDeployment", description = "A deployment of the Data Job")
```

Comment:
Do we need these annotations?

Reply:
I have tested removing these annotations, it does not impact response format. Done. thanks!